elastic-sw-client/delete.py

```diff
@@ -1,12 +1,10 @@
 from elasticsearch import Elasticsearch
-import os
+es = Elasticsearch(['elasticsearch.default.svc.cluster.local:9200'])
```

Comment:
Why use a FQDN? This will break demos if they are going to be set up in a different namespace.
Please consider using this
> Elasticsearch(['elasticsearch.{}:9200'.format(os.environ['NAMESPACE'])])